declarations_site/catalog/templates/static.jinja

```diff
@@ -10,7 +10,9 @@
             </ol> 
         </div>
 
-        <h1 id="page-header"><span>{% block page_header %}{% endblock %}</span></h1>
+        <h1 id="page-header"><span>{% block page_header %}{% endblock %}</span>
+        {{ social("Статична сторінка","http://declarations.com.ua%s" % request.path , add_disqus=True) }}</h1>
```

Comment:
Also, there should be no space before comma. But one after the comma is required.